assets/js/search-utilities.js

```diff
@@ -50,16 +50,32 @@ const index = new FlexSearch.Document({
     ],
     index: ['title', 'content'],
   },
-});
+};
+
+if (window.siteLanguage === 'kor') {
+  indexConfig.encode = (str) => {
+    // not replacing with `str.replace(/[\x00-\x7F]/g, "").split("")` because we
+    // want to mix english words with korean. For example `요소는 npm`
+    return str.split('');
+  };
+}
+
+const index = new FlexSearch.Document(indexConfig);
 
 window.searchIndexReady = false;
 function getBaseUrl() {
   const baseUrl = '{{.Site.BaseURL}}';
-  return baseUrl.endsWith('/') ? baseUrl : baseUrl + '/';
+  return baseUrl.endsWith('/') ? baseUrl : baseUrl.slice(0, -1);
```

Comment:
Why do we remove the last character? If the baseURL is `home/en/search` isn't this going to remove the h from search?